packages/docs/getting-started/reconnecting-to-network.md

```diff
@@ -0,0 +1,148 @@
+# How to reconnect to Baklava after a network reset
+
+If you were running a Validator and Proxy in a previous phase of the `baklava` network, this page contains instructions about how to set up your Validator on the current phase.
```

Comment:
@asaj Should we add an extra sentence here that says: "If you've automated running a Celo validator with a script, please review these and the "Running a Validator" instructions again as there have been a number of small updates and security-related changes."